controller/workitem.go

```diff
@@ -496,7 +496,7 @@ func ConvertJSONAPIToWorkItem(ctx context.Context, appl application.Application,
 			}
 			if err := appl.Areas().CheckExists(ctx, areaUUID.String()); err != nil {
 				switch reflect.TypeOf(err) {
-				case reflect.TypeOf(&goa.ErrorResponse{}):
+				case reflect.TypeOf(&errors.NotFoundError{}):
```

Comment:
It should be possible to call `errs.Cause(err)` to get the cause for an error and that would be the NotfoundError directly. So no need for reflection, right?